ex26.py

```diff
@@ -0,0 +1,39 @@
+game = [[1, 2, 0],
+	    [2, 1, 0],
+	    [2, 1, 1]]
+
+def get_winner(board):
+    #for rows
```

Comment:
Add proper docstrings to the code.Doc strings helps to understand the reader/ developer the purpose of the functions. Applicable wherever you have declared the functions. Also, name the functions properly. Users should be able to understand the purpose of the function by its name itself.